velox/tpcds/gen/dsdgen/dist.cpp

```diff
@@ -371,7 +371,6 @@ int load_dists() {
   read_ptr = tpcds_idx + tpcds_idx_len - (entry_count * IDX_SIZE);
   for (int i = 0; i < entry_count; i++) {
     d_idx_t entry;
-    memset(&entry, 0, sizeof(const d_idx_t));
```

Comment:
There's difference between new and old code, for an example if someone compares two different entries with memcmp it won't work now (because there's padding bytes inside d_idx_t which are not initialized by zero).

But I think there's no such code, so it should be ok